ding/policy/ppg.py

```diff
@@ -615,11 +709,15 @@ def learn_aux(self) -> Tuple[torch.Tensor, torch.Tensor, torch.Tensor]:
 class PPGOffPolicy(Policy):
     """
     Overview:
-        Policy class of PPG algorithm.
+        Policy class of PPG algorithm with off-policy training mode. We implement off-policy PPG with two buffers \
```

Comment:
"We implement off-policy PPG with two buffers with different data use constraint (max_use), which policy buffer offers data for policy phase while value buffer provides auxiliary phase's data." -> "Off-policy PPG contains two different data max_use buffers.  The policy buffer offers data for policy phase , while the value buffer provides auxiliary phase's data."